functions/exercise4/script.js

```diff
@@ -1 +1,10 @@
-"use strict";
\ No newline at end of file
+"use strict";
+
+function validateAsPhraseConsistsOfLettersOnly(phrase) {
+    if (/^[a-zA-Z]*$/g.test(phrase)) {
```

Comment:
It is a duplication of what you have in line 7., but it is only reverted. Why can't you simply skip this if and return true in else statement?